boot/makebootable_test.go

```diff
@@ -591,6 +627,148 @@ version: 5.0
 	c.Assert(err, ErrorMatches, `cannot install managed bootloader assets: internal error: no boot asset for "grub.cfg"`)
 }
 
+func (s *makeBootable20Suite) TestMakeBootable20RunModeSealKeyErr(c *C) {
+	bootloader.Force(nil)
+
+	model := makeMockUC20Model()
```

Comment:
This looks like there is some duplicated code with other similar tests in the setup. But probably followup material at this point.